### ISSUE TITLE
Emitted index.html doesn't work with historyApiFallback

### ISSUE THREAD

**Issue by nstepien** [PepsRyuu/nollup#136]
`.nolluprc`:
```json
{
  "port": 5555,
  "contentBase": "dist",
  "historyApiFallback": true
}
```

In my `rollup.config.js` I have:
```js
import html from '@rollup/plugin-html';

export default {
  // ...
  plugins: [
    // ...
    html({ template }), // `template` is a function returning a custom html page
    // ...
  ]
}
```

This emits the `index.html` file in memory when running nollup, so I can access it via `http://localhost:5555/index.html`, but I get `Cannot GET /` when I load `http://localhost:5555/`.

**Comment by PepsRyuu** [comment]
Released in ```0.13.7```

**Comment by nstepien** [comment]
It's working now, thanks for the quick week-end fix!
One small issue remains: nollup prefers the on-disk `index.html` over the in-memory `index.html` 🤔

**Comment by PepsRyuu** [comment]
Hmmm, that's weird. It should be coming first in the Express logic. I'll check it out tomorrow!

**Comment by PepsRyuu** [comment]
Spotted the problem. ```express.static``` was capturing all requests to ```/``` and returning the physical ```index.html```, so the history API fallback never triggers. Also in the situation where history API fallback is not used, the dev middleware didn't check if index was an emitted asset. 

Should all be working as intended in the next release! :)

**Comment by PepsRyuu** [comment]
Released in ```0.13.8```

**Comment by nstepien** [comment]
Works perfectly now, thanks!
